scripts/west_commands/run_common.py

```diff
@@ -363,17 +372,18 @@ def dump_context(command, args, unknown_args):
     else:
         cls = None
 
-    if runner_config is None:
+    if not runner_configs:
```

Comment:
Personal opinion, feel free to ignore: Consider `if len(runner_configs) == 0` as it is more explicit and easier to read in my opinion, as you clearly see that its a list being checked.